backend/rates/rates.go

```diff
@@ -122,6 +123,38 @@ func (updater *RateUpdater) Last() map[string]map[string]float64 {
 	return updater.last
 }
 
+// LastForPair returns the conversion rate for the given (coin, fiat) pair. Returns an error if the
+// rates have not been fetched yet.
+func (updater *RateUpdater) LastForPair(coinCode string, fiat string) (float64, error) {
```

Comment:
A bit incosistent: uses different coin code keys compared to the `Last` method.
Also, `coinCode, fiat string` is shorter.

Reply:
I am just converting from coin.Code to CryptoCompare codes, explicitly.

Elsewhere it is done implicitly by using coin.Unit, etc., which I don't like.